ansible/roles/uptime-kuma/templates/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.3'
+services:
+    uptime-kuma:
```

Comment:
This whole compose file should be merge with the NGINX one. No need to run 2 compose instances if NGINX is only managing traffic for `uptime-kuma`